packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.styles.ts

```diff
@@ -0,0 +1,427 @@
+      },
+      {
+        selectors: {
+          [`.${IsFocusVisibleClassName} &:focus`]: {
+            opacity: 1
+          }
+        }
+      }
+    ],
+
+    cellWrapperPadded: [
+      {
+        paddingRight: values.isPaddedMargin + values.cellPadding
+      }
+    ],
+
+    cellIsCheck: [
+      classNames.cellIsCheck,
+      {
+        position: 'relative',
+        padding: 0,
+        margin: 0,
+        display: 'inline-flex',
+        alignItems: 'center'
+      },
+      isAllSelected && {
+        opacity: 1
+      }
+    ],
+    cellIsActionable: [
+      {
+        selectors: {
+          '&:hover': {
+            color: semanticColors.bodyText,
+            background: semanticColors.listHeaderBackgroundHovered
+          },
+          '&:active': {
+            background: semanticColors.listHeaderBackgroundPressed
+          }
+        }
+      }
+    ],
+    cellIsEmpty: [
+      {
+        textOverflow: 'clip'
+      }
+    ],
+    cell: [
+      classNames.cell,
+      getFocusStyle(theme),
+      FontClassNames.small,
+      {
+        color: colors.headerForegroundColor,
+        position: 'relative',
+        display: 'inline-block;',
+        boxSizing: 'border-box',
+        padding: `0 ${values.cellPadding}`,
+        border: 'none',
+        lineHeight: 'inherit',
+        margin: '0',
+        height: values.rowHeight,
+        verticalAlign: 'top',
+        whiteSpace: 'nowrap',
+        textOverflow: 'ellipsis',
+        textAlign: 'left',
+        selectors: {
+          '&:hover $gripperBarVerticalStyle': {
+            display: 'block'
+          },
+          '& $filterChevron': {
+            color: colors.dropdownChevronForegroundColor,
+            paddingLeft: 4,
+            verticalAlign: 'middle'
+          },
+          $collapseButton: {
+            paddingRight: 0
+          }
+        }
+      }
+    ],
+
+    gripperBarVerticalStyle: [
+      {
+        display: 'none',
+        position: 'absolute',
+        textAlign: 'left',
+        color: palette.neutralTertiary,
+        left: 1
+      }
+    ],
+
+    cellSizer: [
+      classNames.cellSizer,
+      focusClear(),
+      {
+        display: 'inline-block',
+        position: 'relative',
+        cursor: 'ew-resize',
+        bottom: 0,
+        top: 0,
+        overflow: 'hidden',
+        height: 'inherit',
+        background: 'transparent',
+        zIndex: 1,
+        width: 16,
+
+        selectors: {
+          '&:after': {
+            content: '',
+            position: 'absolute',
+            top: 0,
+            bottom: 0,
+            width: 1,
+            background: colors.resizerColor,
+            opacity: 0,
+            left: '50%'
+          },
+
+          '&:focus:after': cellSizerFadeInStyles,
+          '&:hover:after': cellSizerFadeInStyles,
+          '&$cellIsResizing:after': {
+            ...cellSizerFadeInStyles,
+            ...{
+              boxShadow: '0 0 5px 0 0 rgba(0, 0, 0, 0.4)'
+            }
```

Comment:
Could this just be an array of objects instead of spreading two objets into one?

Reply:
Apparently not for styles listed inside a "selectors" part...